WalletWasabi.Gui/Controls/WalletExplorer/WalletExplorerViewModel.cs

```diff
@@ -25,6 +28,8 @@ public class WalletExplorerViewModel : ToolViewModel, IActivatableExtension
 		private Dictionary<Wallet, WalletViewModelBase> _walletDictionary;
 
 		public override Location DefaultLocation => Location.Right;
+		public bool IsLurkingWifeMode => _isLurkingWifeMode?.Value ?? false;
```

Comment:
I am pretty sure there is a reason to do this because looks weird. Why is this?

Reply:
@lontivero we dont have access to the lurking wife mode boolean property needed for switching icons in the button so i had to code that in. i use OAPH since it's only one way property anyway

Reply:
It was code moved from WalletViewModel to here... I dont know why it should be like that, _isLurkingWifeMode should never be null.

im not sure I want to change it as its using ObservableToPropertyHelper, (which i discourage the use of) which seems to have magic behaviors!

Reply:
Doesn't look weird for me as we use this pattern like everywhere. We can discuss if we should drop this pattern, but let's keep that discussion out of the scope of this PR.